aip/client-libraries/4210.md

```diff
@@ -88,6 +88,11 @@ methods for each RPC.
 - Additionally, if the classes generated for each service support using OAuth
   and service credentials, they **must** honor the `google.api.oauth_scopes`
   annotation (if it is provided), and use these scopes by default.
+- Services that have set the `deprecated` protobuf option **should** have an
+  equivalent deprecation tag generated in the generated class, with a comment
+  specifying when the service will be removed. This is typically the next major
```

Comment:
> comment specifying when the service will be removed

Is this like...an exact date? or some general comment like "in the next major release"?

Reply:
General comment, since such release dates vary per language and dates are generally hard to determine.

Reply:
Thanks!